benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -291,6 +328,7 @@ int main(int argc, char *argv[])
   bool sort_predicates;
   std::string source_pt_cloud;
   std::string target_pt_cloud;
+  std::vector<std::string> exact_specs;
   // clang-format off
     desc.add_options()
         ( "help", "produce help message" )
```

Comment:
Create a vector of `Spec` here, and then do `bpo::value<std::string>(&specs[0].n_neighbors)`. This way you could avoid all the variables.